packages/react-components/src/components/chart/hooks/useXAxis.ts

```diff
@@ -11,6 +11,17 @@ export const useXAxis = (viewportInMs: ViewportInMs, axis?: ChartAxisOptions): X
       type: 'time' as const,
       axisLabel: {
         hideOverlap: true,
+        textStyle: {
+          color: () => {
```

Comment:
no need for a function, can be `color: #5f6b7a`

Reply:
Resolved